sample-app/src/components/Divider.tsx

```diff
@@ -0,0 +1,19 @@
+import { CompositionMethod, useComposedCssClasses } from "../hooks/useComposedCssClasses"
+
+interface DividerCssClasses {
+  divider?: string
+}
+
+const builtInCssClasses = {
+  divider: 'w-full h-px bg-gray-200 my-4'
+}
+
+interface Props {
+  customCssClasses?: DividerCssClasses,
+  cssCompositionMethod?: CompositionMethod
+}
+
+export default function Divider({ customCssClasses, cssCompositionMethod }: Props) {
```

Comment:
I'm wondering if we need a full-blown Component for this. There's not a whole lot to this Component. The main thing it seems to get us is the ability to share Divider styling across Components. Could we instead define and export the default `divider` styling in either `Facets` or `StaticFilters` and import it from the other Component?

Reply:
Sure, I can move it into StaticFilters